server/core/config/valid/global_cfg.go

```diff
@@ -324,7 +323,7 @@ func (g GlobalCfg) MergeProjectCfg(log logging.Logger, repoID string, proj Proje
 
 // DefaultProjCfg returns the default project config for all projects under the
 // repo with id repoID. It is used when there is no repo config.
-func (g GlobalCfg) DefaultProjCfg(log logging.Logger, repoID string, repoRelDir string, workspace string) MergedProjectCfg {
+func (g GlobalCfg) DefaultProjCfg(repoID string, repoRelDir string, workspace string) MergedProjectCfg {
```

Comment:
The `log` field here was unused so I went ahead and removed it for both the legacy and new code dependencies.